doc/api/async_hooks.md

```diff
@@ -973,6 +973,10 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+While you can create your own implementation on top of the `async_hooks` module,
```

Comment:
I'd say "context storage implementation" for more specificity.

Reply:
I think it's better to use "continuation-local storage" as that's the common name for such APIs in Node.js ecosystem. "Context storage implementation" won't give users much in terms of familiarity.

Reply:
To me, "continuation local storage" seems a bit more of an implementation detail name and more just known to the people that are already familiar with the concept, whereas "context storage" feels somewhat clearer. I don't have a strong opinion either way though. Just feel like it'd be good to be a bit more descriptive about what we are referring to here.

Reply:
"Context storage" does not bring any associations with CLS, or AsyncLocal (C#), or thread local (C/C++, Java). Any object or a map may be seen as a context storage (i.e. a generic container for data). It should be at least "async context storage" to make thing a bit more narrow.

But as I mentioned before, most Node.js developers are already familiar with CLS APIs (`continuation-local-storage` and `cls-hooked` packages), so referring to this name may be helpful for us. Alternatively, we could avoid mentioning a name for this class of APIs, like it's done right now in the PR.

Reply:
I've a pretty strong opinion here: I believe it's obvious to the reader that `While you can create your own implementation` means `While you can create your own implementation of something like ALS`. I don't think need to clarify and I believe the sentence to be good as it is.